app/home/directives/home-series.component.ts

```diff
@@ -0,0 +1,106 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Observable} from 'rxjs';
+
+import {HalDoc} from '../../shared/cms/haldoc';
+import {StoryModel} from '../../storyedit/models/story.model';
+import {SpinnerComponent} from '../../shared/spinner/spinner.component';
+import {ImageLoaderComponent} from '../../shared/image/image-loader.component';
+import {TimeAgoPipe} from '../../shared/date/timeago.pipe';
+import {HomeStoryComponent} from './home-story.component';
+
+@Component({
+  directives: [SpinnerComponent, ImageLoaderComponent, HomeStoryComponent],
+  pipes: [TimeAgoPipe],
+  selector: 'home-series',
+  styleUrls: ['app/home/directives/home-series.component.css'],
+  template: `
+    <header *ngIf="noseries">
+      <h2>Your Standalone Stories</h2>
+      <a *ngIf="count > -1" class="all" href="#">View All {{count}} &raquo;</a>
+    </header>
+    <header *ngIf="!noseries">
+      <a href="#">
+        <image-loader [src]="logo"></image-loader>
+      </a>
+      <p class="count">{{count}} Stories</p>
+      <h1><a href="#">{{title}}</a></h1>
+      <p class="updated">Last updated {{updated | timeago}}</p>
+    </header>
+    <div class="story-list">
+      <home-story *ngFor="let s of stories" [story]="s"></home-story>
+      <div *ngFor="let l of storyLoaders" class="story"><spinner></spinner></div>
+    </div>
+  `
+})
+
+export class HomeSeriesComponent implements OnInit {
+
+  PER_ROW = 5;
+
+  @Input() series: HalDoc;
+  @Input() auth: HalDoc;
+  @Input() noseries: boolean;
+  @Input() rows: number = 1;
+
+  logo: string;
+  count: number = -1;
+  title: string;
+  updated: Date;
+  stories: StoryModel[];
+  storyLoaders: boolean[];
+
+  ngOnInit() {
+    if (this.noseries) {
+      this.loadStandaloneStories();
+    } else {
+      this.loadSeriesStories();
+    }
+  }
+
+  loadSeriesStories() {
+    this.title = this.series['title'];
+    this.count = this.series.count('prx:stories');
+    this.updated = new Date(this.series['updatedAt']);
+    this.series.follow('prx:image').subscribe(
+      img => this.logo = img.expand('enclosure'),
+      err => this.logo = null
```

Comment:
not in this PR, but would be good to impl some default (perhaps you have and I haven't noticed it yet)